prow/config.yaml

```diff
@@ -12,6 +12,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+tide:
+  merge_method:
+    GoogleCloudPlatform/osconfig: squash
+
+  target_url: https://prow.gflocks.com/tide
+
+  queries:
+    - repos:
+      - GoogleCloudPlatform/osconfig
+      labels:
+        - lgtm
+        - approved
+      missingLabels:
+        - do-not-merge
+        - do-not-merge/hold
+        - do-not-merge/work-in-progress
+        - needs-rebase
```

Comment:
we don't have a tide deployment here yet, you can add that to cluster.yaml

Reply:
Sure thing